src/index.html

```diff
@@ -1,17 +1,142 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" >
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      class="form"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="registrationForm"
+    >
+      <fieldset class="form-section personalInformation">
+        <legend class="title-form-section">Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input type="text" name="surname" autocomplete="off">
```

Comment:
Check it everywhere, please 
<img width="1010" alt="image" src="https://user-images.githubusercontent.com/99403198/221538565-38a5ed18-8369-434c-9a26-61bc5a6425cb.png">
```suggestion
          <input 
            type="text" 
            name="surname" 
            autocomplete="off"
          >
```